client/src/components/ProfileCertificationCard/ProfileCertificationCard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
Friendly reminder to get rid of those React imports since Vite doesn't need them.